docs/getting_started.md

```diff
@@ -12,17 +12,17 @@ This will be the first screen displayed.
 
 ## DietPi-Launcher
 
-Once you run `dietpi-launcher` you could see all available DietPi tools. It provides a quick way to run any the DietPi tools: from installing **DietPi Optimised software** to simple configure your device, from enabling services to start to backup your installation and so on.
+Once you run `dietpi-launcher` you could see all available DietPi tools. It provides a quick way to run any of the DietPi tools: Installing **DietPi optimized software items** to simple configure your device, enabling services to start to backup your installation and so on.
```

Comment:
The "**from** A **to** B, **from** C **to** D" was used intentionally. When removing "from", the "to" is misplaced in the sentence and would require a whole sentence rewrite then, like an "A, B, C and D" style?

Reply:
Thx. Changed back.